src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -312,6 +305,28 @@ function MoneyRequestPreviewContent({
                                                 </Text>
                                             )}
                                         </View>
+                                        {isScanning && (
+                                            <View style={[styles.flexRow, styles.alignItemsCenter, styles.mt2]}>
+                                                <Icon
+                                                    src={ReceiptScan}
+                                                    height={variables.iconSizeExtraSmall}
+                                                    width={variables.iconSizeExtraSmall}
+                                                    fill={theme.textSupporting}
+                                                />
+                                                <Text style={[styles.textLabel, styles.colorMuted, styles.ml1, styles.amountSplitPadding]}>{translate('iou.receiptScanInProgress')}</Text>
```

Comment:
Is this the preview card view? If so, just want to check that we're using the correct font size for that, cc @dannymcclain 

If I recall, we had a slightly smaller size on the preview card, but we use label on the actual report header.

Reply:
We do indeed use slightly different sizing in the preview vs. the header. Here's the spec reference:

![image](https://github.com/Expensify/App/assets/7516624/dbbf33ea-1654-40e8-b0fa-cadd1dd130bf)